input/docs/handbook/data-binding/value-converters.md

```diff
@@ -1,77 +1,91 @@
 # Value Converters
 
-
-# Links
-* https://github.com/reactiveui/ReactiveUI/blob/master/ReactiveUI/PropertyBinding.cs#L50
-* https://github.com/reactiveui/ReactiveUI/blob/master/ReactiveUI/Xaml/BindingTypeConverters.cs#L24
-
-* http://stackoverflow.com/questions/23592231/how-do-i-register-an-ibindingtypeconverter-in-reactiveui
-* https://github.com/reactiveui/ReactiveUI/commit/7fba662c7308db60cfda9e6fb3331b7cb514f14c
+A value converter should implement the `IBindingTypeConverter` interface. See an example of a globally registered converter type that converts between Boolean and XAML Visibility: https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/Platforms/windows-common/BindingTypeConverters.cs#L31
 
 ## How GetAffinityForObjects works
 
-Return 0 if you can't convert an object and return anything else if it is possible. The largest value (usually 100) ) wins. 
+Return 0 if you can't convert an object and return anything else if it is possible. The largest value wins. 
 
-    public int GetAffinityForObjects(Type fromType, Type toType)
+```csharp
+public int GetAffinityForObjects(Type fromType, Type toType)
+{
+    if (fromType == typeof(string))
     {
-        if (fromType == typeof(string))
-        {
-            return 100; // any number other than 0 signifies conversion is possible.
-        }
-        return 0;
+        return 100; // any number other than 0 signifies conversion is possible.
     }
+    return 0;
+}
+```
 
 ## How TryConvert works
 
-    public bool TryConvert(object from, Type toType, object conversionHint, out object result)
+TryConvert function takes four arguments. The first one is value we are trying to convert, the second one is type to which we are trying to convert that value. `conversionHint` is like a converter parameter, and `result` is where we need to store the conversion result. TryConvert function should return `true` if conversion succeeds and `false` if conversion fails.
+
+```csharp
+public bool TryConvert(object from, Type toType, object conversionHint, out object result)
+{
+    try
     {
-        try
-        {
-            result = !String.IsNullOrWhiteSpace(from);
-        }
-        catch (Exception ex)
-        {
-            this.Log().WarnException("Couldn't convert object to type: " + toType, ex);
-            result = null;
-            return false;
-        }
-    
-        return true;
+        result = !string.IsNullOrWhiteSpace((string)from);
+    }
+    catch (Exception ex)
+    {
+        this.Log().WarnException("Couldn't convert object to type: " + toType, ex);
+        result = null;
+        return false;
     }
 
+    return true;
+}
+```
+
 ## Registration
 
-    Locator.CurrentMutable.RegisterConstant(
-        new MyCoolTypeConverter(), typeof(IBindingTypeConverter));
+You need to register your custom converter using Splat Locator so ReactiveUI can use it.
```

Comment:
You can use the converter from memory for once off conversion as a parameter to Bind() from memory. This is only required if you want to globally handle conversion etc